src/engine/test/source/builder/builders/opBuilderHelperStringFromArray_test.cpp

```diff
@@ -0,0 +1,134 @@
+/* Copyright (C) 2015-2021, Wazuh Inc.
+ * All rights reserved.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#include <any>
+#include <gtest/gtest.h>
+#include <vector>
+
+#include <baseTypes.hpp>
+
+#include "opBuilderHelperMap.hpp"
+
+using namespace base;
+namespace bld = builder::internals::builders;
```

Comment:
```suggestion
using builder::internals::builders;
```
And please remove every `bld::` occurrences.

Reply:
same as above